tests/framework/defs.py

```diff
@@ -20,3 +22,15 @@
 """Relative path to the location of the kernel file."""
 MICROVM_FSFILES_RELPATH = 'fsfiles/'
 """Relative path to the location of the filesystems."""
+SPEC_S3_BUCKET = 'spec.ccfc.min'
+"""The s3 bucket that holds global Firecracker specifications."""
+DEFAULT_TEST_IMAGES_S3_BUCKET = 'spec.ccfc.min'
+"""The default s3 bucket that holds Firecracker microvm test images."""
+ENV_TEST_IMAGES_S3_BUCKET = 'TEST_MICROVM_IMAGES_S3_BUCKET'
+"""Environment variable for configuring the test microvm s3 bucket.
+If variable exists in `os.environ`, its value will be used as the s3 bucket
+for microvm test images.
+"""
+SCRIPT_FOLDER = os.path.dirname(os.path.realpath(__file__))
```

Comment:
This variable definition is not robust as it references a different path than before, introducing breaking changes in the CI suite (e.g. gitlint test).